docker/base-images/base-server.Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN mkdir -p /xsrc && \
     rm -rf /xsrc
 
 ##### base-server target #####
-FROM alpine:3.19 AS base-server
```

Comment:
We want to know exact alpine version which was used for base image. Please revert this.